tests/run/i1692.scala

```diff
@@ -51,9 +55,6 @@ class LazyNotNullable {
     lazy val l8 = h
   }
 
-  private[this] val i = "I"
-  // not nullable because try is lifted, so i is used outside lazy val initializer
-  lazy val l9 = try i catch { case e: Exception => () } 
```

Comment:
Shouldn't we keep this test and add the other as a new test? Or this the behavior of this one change?